src/Main.java

```diff
@@ -51,39 +53,46 @@ public static void main(String[] args) {
         int cash = 15000;
         int i = 0;
         int gesamt = 0;
-        for (; gesamt < 12_000_000; i++) {
-            gesamt = gesamt + ((gesamt * 7 / 100) / 12);
+        int ganz = 12_000_000;
+        int percent = 7;
+        for (; gesamt < ganz; i++) {
+            gesamt = gesamt + gesamt * percent / 100;
             gesamt = gesamt + cash;
             if (i % 6 == 0) {
                 System.out.println("Месяц " + i + " итого " + gesamt);
             }
         }
 
         System.out.println("Aufgabe 6");
-        int savingsPeriodYear = 9;
-        int contribution = 15000;
-        int bankInterest = 7;
-        for (int year = 1; year <= years; year++) {
-            bankInterest = contribution * bankInterest;
-            if (savingsPeriodYear % 2 == 0) {
-                System.out.println("Сумма накоплениий " + year + " лет " + contribution);
+        percent = 7;
+        cash = 15_000;
+        int monat = 1;
+        int monate = 9 * 12;
+        while (monat <= monate) {
+            cash = cash * (1 + percent);
+            if (monat % 6 == 0) {
+                System.out.println("Сумма накоплениий " + monat + " лет " + cash + " рублей");
             }
+            monat++;
         }
-// совсем не понимаю это задание, нужно конкретное объяснение..Конспект к уроку не поняетен,для меня как новичка
+
         System.out.println("Aufgabe 7");
-        int day = 5;
-        int date = 3;
-        for (int dayNumber = 5; dayNumber <= 31; dayNumber = date++) {
-        if (dayNumber % 7 == 0) {
-            System.out.println("Сегодня пятница " + date + "-е, число. Необходимо подготовить отчёт.");
-            continue;
+        int friday = 2;
+        for (int day = 0; day <= 31; day++) {
```

Comment:
```suggestion
        for (int day = 1; day <= 31; day++) {
```
дни нумеруем с 1, а не с 0